doc/release/2021-08-releases.rst

```diff
@@ -262,10 +281,78 @@ SQL
     In addition, the number of arguments is now always checked during parsing
     (:tarantool-release:`2.10.0-beta1`, :tarantool-issue:`6105`).
 
+-   **[Breaking change]** ``NUMBER`` data type  (:tarantool-release:`2.10.0-beta1`)
+    Arithmetic (``+ * - / % & | ~``) operations and bit-shift operations (``>> <<``) are now illegal.
+
+    Example:
+
+    ..  code-block:: none
+
+        SELECT number_column + 1 was legal in version 2.8.
+        Now it will cause an error.
+
+
+-   **[Breaking change]** ``SCALAR`` data type  (:tarantool-release:`2.10.0-beta1`)
+    Arithmetic (``+ * - / % & | ~``) operations and bit-shift operations (``>> <<``) are now illegal.
+    Concatenation (``||``) operations are now illegal.
+    Values in ``SCALAR`` columns now have data type ``SCALAR``, not the value's data type.
+
+    Example:
+
+    ..  code-block:: none
+
+        TYPEOF(CAST(1 AS SCALAR)) was INTEGER in version 2.8.
+        Now it will be SCALAR.
+
+-   **[Breaking change]** Arithmetic operators must now have numeric operands.
+    String operands are illegal.
+
+    Example:
+
+    ..  code-block:: none
+
+        SELECT 1 + '1' was 2 in version 2.8.
+        Now it will cause an error.
+
+
+-   **[Breaking change]** Operations on SCALAR columns
+    Since the type of a value no longer determines whether an operation is valid,
+    comparisons and functions which require a specific type will no longer work.
+
+    Example:
+
+    ..  code-block:: none
+
+        if table T has only one row with a scalar column containing 'a':
+        UPPER(scalar_column) was 'A' in version 2.8.
+        Now it will cause an error.
+
+
+-   **[Breaking change]**: The ``HEX()`` function (:tarantool-release:`2.10.0-beta1`)
+    ``STRING`` arguments are no longer acceptable; only ``VARBINARY`` arguments are allowed.
+
+    Example:
+
+    ..  code-block:: none
+
+        HEX('a') was '41' in version 2.8.
+        Now it will cause an error.
+
+-   **[Breaking change]** The ``POSITION()`` function (:tarantool-release:`2.10.0-beta1`)
```

Comment:
-   **[Breaking change]**: The ``POSITION()`` function (:tarantool-release:`2.10.0-beta1`